static/sass/styles.scss

```diff
@@ -479,3 +479,9 @@ summary {
     justify-content: flex-start;
   }
 }
+
+// XXX - vanilla issue to add this variant https://github.com/canonical-web-and-design/vanilla-framework/issues/2729
+.p-pagination--centered {
```

Comment:
You could also achieve this by adding `.u-align--center` on the `ol` element itself, removing the need for this modifier class - my instinct initially was to add that class to the `ol`s parent element, which makes me wonder if this suggestion is the best approach.

Reply:
thanks - I've updated it